cypress/integration/basic/role_assignment_deleted.spec.js

```diff
@@ -0,0 +1,73 @@
+const BOOKMARKED_PERSON = 'Sadaf Arshad';
+
+beforeEach(() => {
+  Cypress.Cookies.preserveOnce('_DSS-RM_session');
+});
+
+describe('Test that role assignments can be made', () => {
+  it('Bookmark a person', () => {
+    cy.visit('/applications');
+
+    cy.get('input#search_sidebar.input-large.search-query')
+    .type(BOOKMARKED_PERSON, {delay: 100})
+    .should('have.value', BOOKMARKED_PERSON)
+    .type('{enter}');
+  });
+
+  it('Highlight a role', () => {
+    cy.get('div.role:first a').click({force: true});
+  });
+
+  it('Click on a favorite that is not assigned to the role',() => {
```

Comment:
This test failed for me:

CypressError: Timed out retrying: cy.click() failed because this element is not visible:

<li class="person" style="cursor: default;">...</li>

This element '<li.person>' is not visible because it has an effective width and height of: '0 x 0' pixels.

Fix this problem, or use {force: true} to disable error checking.

https://on.cypress.io/element-cannot-be-interacted-with